### PR TITLE
refactor: repository is renamed from namics to merkle

### PR DESCRIPTION
**Pull request by jMergenthal**
Note: additional needed changes in regards to the renaming of the repository from `eslint-config-nacmics` to `eslint-config` are added in this pull request. Therefore this PR should be merged together with the renaming of the repository itself.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# EditorConfig. Frontend. Default. Namics.
+# EditorConfig. Frontend. Default. Mekrle.
```
Comment (danielkoch) [inline .editorconfig:1]: Typo?

Comment (jMergenthal) [reply .editorconfig]: fixed

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# ESLint config Namics
+# ESLint config
 
-[![Build Status](https://github.com/namics/eslint-config-namics/workflows/ci/badge.svg)](https://github.com/namics/eslint-config-namics/actions)
+[![Build Status](https://github.com/merkle-open/eslint-config/workflows/ci/badge.svg)](https://github.com/merkle-open/eslint-config/actions)
 [![npm](https://img.shields.io/npm/v/@namics/eslint-config.svg)](https://www.npmjs.com/package/@namics/eslint-config)
-[![Codestyle](https://img.shields.io/badge/codestyle-namics-green.svg)](https://github.com/namics/eslint-config-namics)
```
Comment (danielkoch) [inline README.md:5]: Please use: https://img.shields.io/badge/codestyle-merkle-green.svg

Comment (danielkoch) [reply README.md:5]: Changed.

**Review by ernscht — APPROVED** [review]
great

**Review by danielkoch — APPROVED** [review]

**Pull request merged**